input_output/graphical_interface.py

```diff
@@ -143,39 +215,87 @@ def get_copy_captured_image(self, resolution):
             )  # linear is faster than cubic
         else:
             raise Exception("original frame is not available")
+    
+    def draw_indicator(self, probabilities):
+        """
+        wrapper of draw_indic
+        """
+        self.is_present_original_frame = False
+        #draw_indic(self.frame, probabilities, self.snapshot, self.font, self.font_scale)
+        draw_indic2(self.frame, probabilities, self.snapshot, self.font, self.font_scale, self.font_thickness)
+
+    def draw_headband(self, under_band = 1):
+        """
+        parameters :
+            under_band (float) : permit to draw a band under the headband : 1 (default) : draw the headband / 1.75 : draw the headband & an underband
```

Comment:
use a boolean if only two state is supported / discribe what the intermiate value do